### ISSUE TITLE
dj webcam project

### ISSUE THREAD

**Issue by silky** [hahayes/hahayes-projects#34]
you can watch a dj play music - http://www.youtube.com/watch?v=Q692lHFaLVM

but the dj can't watch you enjoying the music.

imagine if there was a website where the dj could also look at the people getting funky to their live mix. for the dj, it could just randomly show 20 or so webcams, and you could get a nice overview of who was getting down. make the experience more interactive.

would be fun to use the site while at work

maybe infact the site is just a big screen of such webcams, so you can see 30 or so peoples webcams getting funky, as even a listener.

(this idea needs a name)

**Comment by mobeets** [comment]
yeah that would be fun. if you had a full festival's worth of webcams, you could have some sort of motion detection thing find the most active viewers and highlight them, to keep up the performer's energy.

**Comment by mobeets** [comment]
also reminds me of this: http://blog.sfgate.com/techchron/2013/01/31/an-ipad-telepresence-robot-creates-a-virtual-you/

apparently robotics conferences these days have attendees using these things. would be pretty fun i think!
